BeVolt/Drivers/src/LTC6811.c

```diff
@@ -75,7 +75,185 @@ Copyright 2017 Linear Technology Corp. (LTC)
 /*********************************************************/
 /*** Code that was added by UTSVT. ***/
 /*********************************************************/
```

Comment:
Are these functions meant to be local to the file? If so, `static` should be applied. If not, they should be renamed to match the style guide.